examples/data-streams/index.ts

```diff
@@ -0,0 +1,77 @@
+import { type RemoteParticipant, Room, RoomEvent, type TextStreamReader } from '@livekit/rtc-node';
+import { config } from 'dotenv';
+import { AccessToken } from 'livekit-server-sdk';
+
+config({ path: '.env.local', override: false });
+const LIVEKIT_API_KEY = process.env.LIVEKIT_API_KEY;
+const LIVEKIT_API_SECRET = process.env.LIVEKIT_API_SECRET;
+const LIVEKIT_URL = process.env.LIVEKIT_URL;
+if (!LIVEKIT_API_KEY || !LIVEKIT_API_SECRET || !LIVEKIT_URL) {
+  throw new Error('Missing required environment variables. Please check your .env.local file.');
+}
+
+const greetParticipant = async (room: Room, recipient: RemoteParticipant) => {
+  const greeting = 'Hi this is just a text sample';
+  const streamWriter = await room.localParticipant?.streamText({
+    destinationIdentities: [recipient.identity],
+  });
+
+  for (const c of greeting) {
+    await streamWriter?.write(c);
+  }
+
+  streamWriter?.close();
+};
+
+const sendFile = async (room: Room, recipient: RemoteParticipant) => {
+  console.log('sending file');
+  await room.localParticipant?.sendFile('./assets/maybemexico.png', {
+    destinationIdentities: [recipient.identity],
+    fileName: 'mex',
+    mimeType: 'image/png',
+  });
```

Comment:
i'd add `topic` here as well, and maybe in the other spot to, just to better show how to to use it.